src/sage.ts

```diff
@@ -54,15 +54,25 @@ async function main() {
 
 	bot.once('ready', async () => {
 		// I'm mad about this - Josh </3
-		const team = (await bot.application.fetch()).owner as Team;
-		setBotmasterPerms(team.members.map(value => {
-			const permData: ApplicationCommandPermissionData = {
-				id: value.id,
+		// Don't worry Josh, I fix - Ben <3
+		const owners = (await bot.application.fetch()).owner; // as Team;
```

Comment:
Remove comment

Reply:
IDK, looks like a piece of history to me.
![image](https://user-images.githubusercontent.com/34194692/193427242-0ced7fee-d8d2-4cf1-8f0d-6f65fefd4eec.png)

Reply:
I was talking about `// as Team;`

Reply:
Oh lol. Yeah will do.

Reply:
Done in [4e922a5](https://github.com/ud-cis-discord/Sage/pull/295/commits/4e922a58fc9e02ddce57300f3fe2ba72533834cf)